From 80e1176c95ca6dbbc89e79b4869937b735e0f269 Mon Sep 17 00:00:00 2001
From: Greg Hudson <ghudson@mit.edu>
Date: Sat, 25 May 2013 14:35:41 -0400
Subject: Don't warn or error on variadic macros

Our portability assumptions now allow the use of variadic macros, so
don't warn or error on them.
---
 src/aclocal.m4 |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/aclocal.m4 b/src/aclocal.m4
index 2c17e46..7504b97 100644
--- a/src/aclocal.m4
+++ b/src/aclocal.m4
@@ -538,7 +538,7 @@ if test "$GCC" = yes ; then
     #
     # We're currently targeting C89+, not C99, so disallow some
     # constructs.
-    for flag in declaration-after-statement variadic-macros ; do
+    for flag in declaration-after-statement ; do
       TRY_WARN_CC_FLAG(-Werror=$flag)
       if test "$flag_supported" = no; then
         TRY_WARN_CC_FLAG(-W$flag)
